okio/jvm/build.gradle

```diff
@@ -23,7 +27,21 @@ configurations {
   baseline
 }
 
+jmhJar {
```

Comment:
Struggled for a long time and eventually added this. I’m not sure if this is what makes it work, but it appears to make it work.